mantidproject/mantid
# Indirect OSIRIS Diffraction - Add OSIRIS_GSS_Parameter file to instrument dir

Added the `.prm` file to the instrument directory. This file will be used along side the GSS Output file for analysis in other software.

**To test:**
- Ensure the file is in the instrument directory with the `.prm` extension
- Ensure all tests pass

Fixes #15924 .

[Release Notes](https://github.com/mantidproject/mantid/blob/15924_IndirectOsiDiff_Additional_GSS_output_file/docs/source/release/v3.7.0/indirect_inelastic.rst#improvements)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

Refs #15924
